com.basiomeuspuga.Lector.json

```diff
@@ -0,0 +1,143 @@
+{
+	"app-id":"com.basiomeuspuga.Lector",
+	"runtime":"org.kde.Platform",
+	"runtime-version":"5.12",
+	"sdk":"org.kde.Sdk",
+	"command":"lector",
+	"rename-icon":"Lector",
+	"rename-desktop-file":"lector.desktop",
+	"rename-appdata-file":"lector.appdata.xml",
+	"finish-args":[
+		"--filesystem=host",
```

Comment:
what do you need host permissions for? That seems uncomfortably broad.

Reply:
I imagine books can be located anywhere in the filesystem. I myself keep my books stored in an NTFS filesystem shared between Windows and Linux. That said, is it better to go with `"--filesystem=host:ro"`?

Reply:
It would be better to use xdg-desktop-portal but I'm not actually sure what it takes for Qt. @aleixpol any pointers?

Reply:
That I know of, it's not possible to open directories using portals.